.github/workflows/release.yml

```diff
@@ -12,8 +12,11 @@ jobs:
         uses: actions/checkout@v2
       - name: Build
         run: make release
+        working-directory: Packages/CatbirdApp
       - name: Archive
-        run: zip -r catbird.zip catbird start.sh stop.sh LICENSE Public Resources Sources/CatbirdAPI
+        run: |
+          cd ..
```

Comment:
working-directory affects only the current step?

Reply:
It seems, yes. I will remove this line of code.